keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -365,16 +387,12 @@ def call(self, images, training=None):
         anchors = self.anchor_generator(image_shape=image_shape)
         rois, feature_map, _, _ = self._call_rpn(images, anchors, training=training)
         box_pred, cls_pred = self._call_rcnn(rois, feature_map)
-        if not training:
-            # box_pred is on "center_yxhw" format, convert to target format.
-            box_pred = _decode_deltas_to_boxes(
-                anchors=rois,
-                boxes_delta=box_pred,
-                anchor_format="yxyx",
-                box_format=self.bounding_box_format,
-                variance=[0.1, 0.1, 0.2, 0.2],
-            )
-        return box_pred, cls_pred
+
+        if training:
+            return box_pred, cls_pred
+
+        # Return anchor boxes of interest during evaluation
+        return box_pred, cls_pred, rois
```

Comment:
any reason that we don't need `_decode_delta_to_boxes` anymore?

Reply:
Decoding now happens in the `prediction_decoder`, so we don't need to call it here.